crates/schema/src/schema.rs

```diff
@@ -1008,3 +1041,44 @@ impl Schema for ConstraintSchema {
         Ok(())
     }
 }
+
+/// A struct representing the schema of a row-level security policy.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct RowLevelSecuritySchema {
+    pub row_level_security_id: RowLevelSecurityId,
+    pub row_level_security_name: Box<str>,
```

Comment:
Again I think we can remove the name field. Do we even need the primary key column `row_level_security_id`?

Reply:
There can be multiple RLS queries per table. But given a table, will it ever be necessary to ask for a specific RLS query? If that's not really a plausible use case, then an separate id field seems redundant, as does the name.

Reply:
Chatting with Tyler and Mazdak, we had decided we wanted all entities in the database to have names so that they could be referred to and queried from the system tables. I had initially tried to get rid of all the names, since they were redundant, but was overruled. The intent is now that all entities in the system tables have names so that they can be referred to by the user, even if autogenerated.